aea/crypto/cosmos.py

```diff
@@ -820,6 +820,43 @@ def get_contract_instance(
         # Instance object not available for cosmwasm
         return None
 
+    @staticmethod
+    def get_last_code_id():
+        """
+        Uses wasmcli to get ID of latest deployed .wasm bytecode
+
+        :return: code id of last deployed .wasm bytecode
+        """
+
+        command = ["wasmcli", "query", "wasm", "list-code"]
+
+        stdout, _ = subprocess.Popen(  # nosec
+            command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
+        ).communicate()
+
+        json_out = json.loads(stdout.decode("ascii"))
```

Comment:
You have the exact same code in multiple places here; looks like a private helper method is needed which implements the actual subprocess call.